invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentPanel/DocumentPanelMobile.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { Grid, Header } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DocumentCirculation } from '../DocumentCirculation';
+
+import {
+  ILSHeaderPlaceholder,
+  ILSParagraphPlaceholder,
+} from '../../../../../../common/ILSPlaceholder';
+import {
+  DocumentAbstract,
+  DocumentAuthors,
+  DocumentCover,
+  DocumentTags,
+  DocumentTitle,
+} from '../DocumentMetadata/components';
+
+export default class DocumentPanelMobile extends Component {
+  constructor(props) {
+    super(props);
+    this.document = props.documentDetails;
+  }
+
+  render() {
+    const { isLoading } = this.props;
+    return (
+      <>
+        <div
+          className="document-panel"
+          data-test={this.document.metadata ? this.document.metadata.pid : 0}
```

Comment:
Not just here but why do we name this as `data-test` rather than `data-pid` or similar?

Reply:
because we use this specifically for testing

Reply:
I consider it noise when we throw data-test attributes in our code, but that might be just me.

Reply:
for me it is very clear that is for test :) but no strong preference :)